jellyfinclient/src/actions/ApiClient.js

```diff
@@ -0,0 +1,11 @@
+import ApiClient from 'jellyfin-apiclient/dist/apiclient';
+
+const JFInterface = {
```

Comment:
Yes I agree. I thought that using redux for the API client would just make the code more complicated but it will come in handy later I guess. I will fix this

Reply:
Well I just started rewriting the redux store a little and found out that when I store an instance of ApiClient in the redux store, redux removes all functions from the class. This is probably because those are not serializable. So I would leave the ApiClient outside of the redux store and reinitalize it everytime on startup. what do you think?

Reply:
With thunk you can add dependency:

https://github.com/reduxjs/redux-thunk#injecting-a-custom-argument

You just use the withExtraArgument and every thunk-action will have this new argument

Reply:
The API Client gets stored in redux now